### PR TITLE
Unknown subcommand `rvm gemset clear`

### PR DESCRIPTION
**Pull request by pkuczynski**
Fixes rvm/rvm#4004

### CODE REVIEW

**Comment by mpapis** [comment]
could we link the file to http://rvm.io/gemsets/using and add the `@default` there?

**Comment by pkuczynski** [comment]
Not sure what you meant by "linking" but I updated `using.md` - is this what you wanted?

**Comment by mpapis** [comment]
```bash
ls -l ./content/integration/cron.md
lrwxrwxrwx 1 mpapis mpapis 21 sty 13  2016 ./content/integration/cron.md -> ../deployment/cron.md
```
this way the links will still work if it was added somewhere, but will not require extra manual work

**Comment by pkuczynski** [comment]
Ah, now I see. We should rather return 302 instead of displaying duplicated content, don't you think? :)

**Comment by pkuczynski** [comment]
I added the link, but I think we should rather redirect then display duplicated content. Maybe we should implement https://github.com/gjtorikian/nanoc-redirector? @mpapis what you think?

File: content/gemsets/using.haml [hunk 1]
```diff
@@ -27,6 +27,13 @@ title: "'rvm gemset use' - Using gemsets"
     $ rvm gemset use teddy
 
 %p
+  To stop using the custom gemset (and switch to the default one) use one of the following:
+%pre.code
+  :preserve
+    $ rvm gemset use default
```
Comment (mpapis) [inline content/gemsets/using.haml]: when presenting two or more possibilities I usually add `# OR:` at the end of line

**Comment by pkuczynski** [comment]
`# OR:` added

**Comment by pkuczynski** [comment]
I will create separate PR implementing `nanoc-redirector`

**Pull request merged**